src/mapgen/mapgen.cpp

```diff
@@ -761,6 +762,19 @@ void MapgenBasic::generateBiomes()
 
 			VoxelArea::add_y(em, vi, -1);
 		}
+		// If no stone surface was detected in this mapchunk column the biomemap
+		// will be empty for this (x, z) position. Add the currently active
+		// biome to the biomemap, or if biome is NULL calculate it for this
+		// position.
+		if (biomemap[index] == BIOME_NONE) {
+			if (biome) {
+				biomemap[index] = biome->index;
+			} else {
+				biome =
+					biomegen->getBiomeAtIndex(index, v3s16(x, node_min.Y, z));
```

Comment:
Indentation looks weird due to github 8 space indent, but is needed to avoid a long line with 4 space indent.

Reply:
It would also be allowed to break the line before it reaches the character limit.

Reply:
Oops, thanks will fix that soon.
An alternative line split may be better too.

Reply:
Done in #7398